### DISCUSSION TITLE
Is there a specific recommended Docker version? (v5.1.4-beta)

### DISCUSSION THREAD

**Discussion by Mentors4EDU** [blockscout/blockscout#7523] [General]
I been having problems with Docker's default version and Erlang trying to do it on Ubuntu 20.04 LTS. Also, been having problems with the manual deployment as well. Any recommendations for [BlockScout v5.1.4-beta](https://github.com/blockscout/blockscout/releases/tag/v5.1.4-beta)?

**Comment by sl1depengwyn** [comment]
What kind of problems are you facing? Could you please specify?
